### ISSUE TITLE
Can your model be used with custom data?

### ISSUE THREAD

**Issue by Mitko88** [EasonChen99/I2D-Loc#3]
I have a point cloud and an image, and I want to check if correct rotation can be found for the image based on your trained mode.

**Comment by EasonChen99** [comment]
As mentioned in the paper, our trained model exhibits a certain degree of generalization to data with diverse camera intrinsic parameters. However, for higher localization accuracy, we recommend fine-tuning the model on your specific dataset to achieve optimal results.

**Comment by jiheddachraoui** [comment]
hello how to update datasets with high image resolution like Boreas?
